canvases/dark-packages/main.dark

```diff
@@ -1,15 +1,21 @@
-type PackageFns = PACKAGE.Darklang.Stdlib.ProgramTypes.PackageFn.T
+type PackageFns =
```

Comment:
TODO: replace with `PACKAGE.Darklang.Stdlib.ProgramTypes.PackageFn.T`

But we still cant do 
```
[<DB>]
type PackageFnsDB = PACKAGE.Darklang.Stdlib.ProgramTypes.PackageFn.T
```
The DB is empty when we do DB.getAll. No visible errors, and also no errors in the logs